src/Microsoft.AspNetCore/CertificateLoader.cs

```diff
@@ -65,23 +121,18 @@ private abstract class CertificateSource
 
         private class CertificateFileSource : CertificateSource
         {
-            private readonly string _password;
-
-            public CertificateFileSource(string password)
-            {
-                _password = password;
-            }
-
             public string Path { get; set; }
 
+            public string Password { get; set; }
+
             public override X509Certificate2 Load()
             {
-                var certificate = TryLoad(X509KeyStorageFlags.DefaultKeySet, out var error)
-                    ?? TryLoad(X509KeyStorageFlags.UserKeySet, out error)
-    #if NETCOREAPP2_0
-                    ?? TryLoad(X509KeyStorageFlags.EphemeralKeySet, out error)
-    #endif
-                    ;
+                Exception error;
+                var certificate =
+#if NETCOREAPP2_0
+                    TryLoad(X509KeyStorageFlags.EphemeralKeySet, out error) ??
+#endif
+                    TryLoad(X509KeyStorageFlags.UserKeySet, out error);
```

Comment:
@Eilon @DamianEdwards @muratg @davidfowl 
We should prefer ephemeral if it is available, which is only available in netcoreapp2.0. I believe that means this package (Microsoft.AspNetCore) needs to target netcoreapp2.0, not netstandard1.3.